public_html/wp-content/plugins/camptix/notice.diff

```diff
@@ -0,0 +1,74 @@
+Index: camptix.php
```

Comment:
guess this was included in the PR by mistake?

Reply:
🤦‍♂️